### DISCUSSION TITLE
SQLITE error

### DISCUSSION THREAD

**Discussion by robbie-berns** [EasyEngine/easyengine#1765] [Support Requests]
Hi. This is a first time with this problem.

Any ee command returns an error: **_Error: Unable to initialize connection to EE sqlite database: could not find driver_**

The server is other working fine. Any suggestions?

**Comment by mrrobot47** [comment]
It seems you are missing `SQLite 3 driver` on your server. What you need is the following:
```bash
apt update && apt install -y sqlite3
```
